src/plugins/intel_npu/src/backend/src/zero_executor.cpp

```diff
@@ -100,6 +83,55 @@ ZeroExecutor::ZeroExecutor(const std::shared_ptr<const ZeroInitStructsHolder>& i
         }
     }
 
+    if (_graph_ddi_table_ext.version() < ZE_GRAPH_EXT_VERSION_1_8) {
+        initilize_graph_through_command_list();
+    } else {
+        ze_graph_properties_2_t properties = {};
+        properties.stype = ZE_STRUCTURE_TYPE_GRAPH_PROPERTIES;
+        _graph_ddi_table_ext.pfnGetProperties2(_graph, &properties);
+
+        switch (properties.initStageRequired) {
```

Comment:
This is a bitfield, the logic should be changed to test for each enum value (e.g. `if (properties.initStageRequired & ZE_GRAPH_STAGE_INITIALIZE)`) and call the appropriate initialization.

Reply:
done, thx